sklearn/ensemble/_voting.py

```diff
@@ -64,7 +64,7 @@ def fit(self, X, y, sample_weight=None):
                              '; got %d weights, %d estimators'
                              % (len(self.weights), len(self.estimators)))
 
-        self.estimators_ = Parallel(n_jobs=self.n_jobs)(
+        self.estimators_ = Parallel(n_jobs=self.n_jobs, verbose=self.verbose)(
                 delayed(_parallel_fit_estimator)(clone(clf), X, y,
```

Comment:
The `_parallel_fit_estimator` doesn't have a `verbose` parameter I think, and so tests would be failing? You can run tests locally with,
```
pytest sklearn/ensemble/
```